README.md

```diff
@@ -332,7 +332,7 @@ helpful discussions on the drand design.
 
 Thanks to [@Bren2010](https://github.com/Bren2010) and
 [@grittygrease](https://github.com/grittygrease) for providing the native
-Golang bn256 implementation and for their help in the design of drand and
+Golang bls12-381 implementation and for their help in the design of drand and
```

Comment:
Is this thank you message still correct? Did grittygrease provide the bls12-381 implementation also?

Reply:
Wasn't meant to catch that one. Rolled back.